src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -29,7 +29,7 @@ object WorkspaceCreate {
     new WorkspaceCreate(
       namespace = wc.namespace,
       name = wc.name,
-      attributes = wc.attributes + (AttributeName("library","published") -> AttributeBoolean(false)),
+      attributes = wc.attributes + (AttributeName("library","published") -> AttributeBoolean(false)) + (AttributeName("library","discoverableByGroups") -> AttributeValueEmptyList),
```

Comment:
Probably not for this PR - just thinking out loud here. We're at the point where there is a set of library attributes that are non-clone-able. We should think about maintaining that set closer to `AttributeName` than in `WorkspaceCreate`.